### ISSUE TITLE
maxspeed:practical - use with conditional restrictions

### ISSUE THREAD

**Issue by RicoZ-OSM** [osmandapp/Osmand#2051]
I would like to extend maxspeed:practical to use "modern" conditional restrictions as described in
   http://wiki.openstreetmap.org/wiki/Conditional_restrictions
so that you could write something like 

```
maxspeed:practical:conditional=60 @ (07:00-9:00);  90 @ (09:00-16:00); 50 @ (16:00-18:00); 70 @ wet
```

Could OsmAnd handle this?

**Comment by pl71** [comment]
OsmAnd could handle almost everything, but [practical] (http://wiki.openstreetmap.org/wiki/Proposed_features/Practical_maxspeed) is a rejected proposal as far as I know.

**Comment by RicoZ-OSM** [comment]
Rejected but nevertheless in use and OsmAnd does use it as far as I can figure out from the code.

The question was - could it handle the conditional restrictions as well?

**Comment by vshcherb** [comment]
So far there is no support of time (!) in routing engine. Even though UI supports maxspeed:conditional but it is not supported by the engine and that's a big difference. So this clearly an issue for both tags
maxspeed:conditional and maxspeed:practical:conditional

**Comment by vshcherb** [comment]
Merged into #1794
